### PR TITLE
Add 9.2.4 to the API dropdown and bump 8.1.0 references to 8.2.0.

### PR DESCRIPTION
**Pull request by tplunket**

### CODE REVIEW

**Comment by tplunket** [comment]
I just started using OpenLayers and noticed the version bump but no UI-supported way to get back to the version I was developing on, so here it is! Also noticed that 8.1.0 was referenced everywhere but not 8.2.0 so made that change too.

File: dist/en/main/apidoc/module-ol_style_expressions.html [hunk 1]
```diff
@@ -2,7 +2,7 @@
 <html lang="en">
   <head>
     <meta charset="UTF-8">
-    <title>OpenLayers v8.1.1-dev API - Module: ol/style/expressions</title>
+    <title>OpenLayers v10.0.1-dev API - Module: ol/style/expressions</title>
```
Comment (MoonE) [inline dist/en/main/apidoc/module-ol_style_expressions.html:5]: This file can be removed, the module no longer exists.

**Review by MoonE — COMMENTED** [review]
Thanks for this.
The source code for the website lives in the https://github.com/openlayers/openlayers repository, so it needs to be changed there, too.

https://github.com/openlayers/openlayers/blob/main/site/layouts/default.hbs
https://github.com/openlayers/openlayers/blob/main/site/src/index.hbs
A PR would be appreciated.

File: dist/en/latest/apidoc/module-ol_style_expressions.html
Comment (tplunket) [inline dist/en/latest/apidoc/module-ol_style_expressions.html:1]: Can this file be removed as well, then?

**Comment by tplunket** [comment]
openlayers/openlayers#16041 for the main repo. Does that end up generating what's necessary for this repo, or are these files all managed manually?

**Comment by ahocevar** [comment]
Fixed with https://github.com/openlayers/openlayers/pull/16041 - these files are auto-generated. Thanks for your efforts, @tplunket and @MoonE .

**Pull request closed without merging**